src/config/_manager.py

```diff
@@ -0,0 +1,32 @@
+"""
+$file.fileName - $shortDescripion  # FIXME template
```

Comment:
[Style] FIXME

Reply:
I argue that a `fixme` is error. will fix